src/interpret.jl

```diff
@@ -21,7 +21,8 @@ end
 sbmlLog(x) = sbmlLog(10, x)
 sbmlLog(base, x) = log(base, x)
 
-sbmlPower(x, y) = x^float(y)
+sbmlPower(x, y::Int) = x^float(y)
```

Comment:
this might be extended to y::Number or at least y::Real, right?
```suggestion
sbmlPower(x, y::Real) = x^float(y)
```

Reply:
I don't think this is necessary, cause it is only negative Ints in the exponent that cause trouble. Or do you know another case where simply x^y does not work?

Reply:
ah yeah, actually what would break there with the negative exponent? can we omit the float() conversion altogether?

I guess it would be much better to actually find and document the reason for the extra `float` there; if the code is like this we'll break it with the next bugfix of something else :D